examples/models/llama2/export_llama_lib.py

```diff
@@ -423,6 +430,10 @@ def _prepare_for_llama_export(modelname: str, args) -> LLMEdgeManager:
             # to get free perf gain.
             transforms.append(replace_sdpa_with_simple_sdpa)
             transforms.append(replace_causal_mask)
+
+    if args.optimized_rotation_path:
+        transforms.append(fuse_layer_norms)
+        transforms.append(get_rotate_model(args.optimized_rotation_path))
```

Comment:
I'm trying to follow, why do we need to do `get_rotate_model` as a transform?

Reply:
Under my understanding, we need to rotate the weight before running ptq flow.
I refer to [this function](https://github.com/facebookresearch/SpinQuant/blob/b3b0e56279ee06ad597d4c54a19784272dafed2a/eval_utils/rotation_utils.py#L123) in spin quant repo.

Reply:
oh I see, so you just got R1/R2 from spin quant directly, instead of getting a new weight check point?

Reply:
Yes that's what I do.
Due to hardware constraint ....... as you known.